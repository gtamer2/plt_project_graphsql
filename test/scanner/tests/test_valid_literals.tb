0
1
2
3
4
5
6
7
8
9
9876543210
88.27