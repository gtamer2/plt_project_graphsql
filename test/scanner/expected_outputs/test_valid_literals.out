LITERAL: 0
LITERAL: 1
LITERAL: 2
LITERAL: 3
LITERAL: 4
LITERAL: 5
LITERAL: 6
LITERAL: 7
LITERAL: 8
LITERAL: 9
LITERAL: 9876543210
LITERAL: 88.27